File: bindings/python/text_search_and_replace.py
import re, easyre

def render(input, selection, op):
	regexp = easyre.easyRE(op['search'])
	
	#TODO inselection param is ignored !!!

	return re.sub(regexp, op['replace'], input, flags = re.IGNORECASE if not op['casesensitive'] else 0)

File: bindings/js/text_search_and_replace.js
var easyRE = require('./easyre.js')

exports.render = function (input, selection, op){
	var re = easyRE.easyRE(op.search);
	try{
		var regexp = new RegExp( re, 'g'+(op.casesensitive?'':'i'));
	}
	catch(e){
		return 'ERROR en el campo de búsqueda';
	}
	
	//TODO inselection param is ignored !!!

	return input.replace(regexp, op.replace);
}
